sgp4/tests.py

```diff
@@ -536,6 +536,7 @@ def run_satellite_against_tcppver(twoline2rv, invoke, expected_errors):
     # output in tcppver.out.
 
     data = get_data(__name__, 'tcppver.out')
+    data = data.replace(b'\r', b'')
```

Comment:
Nice, I'm glad this fix wound up being simple!

Reply:
Great, we keep that. Thanks for the hint.